proto/reaction.proto

```diff
@@ -712,9 +712,7 @@ message ReactionProvenance {
   }
   RecordEvent record_created = 7;
   repeated RecordEvent record_modified = 8;
-  // TODO(ccoley): is it useful to create a unique ID field 
-  // that the centralized database can write to? (I know uniqueness is not
-  // enforceable)
+  // This is a unique ID field that the centralized database will write to
```

Comment:
Super nit: please add a period :)

http://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar

Reply:
Reluctantly done

Reply:
;)